src/js/myboard.js

```diff
@@ -34,17 +34,7 @@ getAppRequestInfo = function() {
 };
 
 additionalCallback = function() {
-    Common.setAppCellUrl();
-
-    Common.setAccessData();
-
-    if (!Common.checkParam()) {
-        // cannot do anything to recover
-        // display a dialog and close the app.
-        return;
-    };
-
-    Common.getAppDataAPI(Common.getTargetUrl(), Common.getToken()).done(function(data) {
+    Common.getAppDataAPI(Common.getBoxUrl(), Common.getToken()).done(function(data) {

```

Comment:
Since Common.getBoxUrl () does not immediately return BoxURL, it can not be used here.
Prepare and use getter.